src/pineko/cli/kfactor.py

```diff
@@ -11,28 +11,28 @@
 @command.command("kfactor")
 @click.argument("grids_folder", type=click.Path(exists=True))
 @click.argument("kfactor_folder", type=click.Path(exists=True))
-@click.argument("yamldb_path", type=click.Path(exists=True))
+@click.argument("yamldb_file", type=click.Path(exists=True))
 @click.argument("target_folder", type=click.Path(exists=True))
-@click.argument("max_as", type=int)
-@click.argument("order_exists", type=bool)
+@click.argument("pto_to_update", type=int)
+@click.option("--order_exists", is_flag=True, help="Owerwrite an existing order.")
```

Comment:
```suggestion
@click.option("--overwrite", is_flag=True, help="Owerwrite an existing order.")
```
maybe? just as we do for `theory`

Reply:
I though the same, but it would be more confusing. `overwrite` is done to overwrite an operation that has been done with the same pineko command, while here the situation is slightly different, you want to overwrite and order that is already present.